### ISSUE TITLE
Resizing window causes app to crash in latest release (v0.10.761.0)

### ISSUE THREAD

**Issue by ev-dev** [microsoft/terminal#4972]
Updated to the latest preview release today (v0.10.761.0)

Immediately noticed that any attempts to resize the window either via mouse drag or keyboard shortcut causes the Terminal app to become unresponsive and crash. 

Same behavior for both normal and admin/elevated launches of Terminal app. 

# Environment
```none
Windows build number: 10.0.19582.0
Windows Terminal version (if applicable): 0.10.761.0

Any other software? No
```

# Steps to reproduce
Either:
- Use mouse to resize the Terminal app window
- Use keyboard shortcuts to maximize or split screen the window (via Win+Up/Right/Left)

# Expected behavior
Normal resizing which broke for me since latest build (v0.9 was resizing fine)

# Actual behavior
Application becomes unresponsive and crashes

**Comment by FractalBoy** [comment]
i'm seeing this as well.

**Comment by bjornmonkey** [comment]
Microsoft Windows [version 10.0.19041.153]

i am seeting this as well, 100% reproduce

**Comment by zadjii-msft** [comment]
Hey for the folks seeing this - could any of you share your `profiles.json`? I think I've got a theory on what's causing this, but I want to make sure.

It might also just be helpful if one of you were able to file feedback and share the link here: /feedback

**Comment by bjornmonkey** [comment]
Please see the feedback: https://aka.ms/AA7rqvs

**Comment by zadjii-msft** [comment]
@bjornmonkey What value do you have `historySize` set to?

**Comment by bjornmonkey** [comment]
> @bjornmonkey What value do you have `historySize` set to?

100,000

**Comment by bjornmonkey** [comment]
i tried reduce my historysize to 10,000, it looks ok now.

**Comment by FractalBoy** [comment]
my history size is also large - 100000000

**Comment by FractalBoy** [comment]
reducing to 10000 also fixes it for me

**Comment by zadjii-msft** [comment]
Aces, that's good to know. Fortunately, the fix is already in PR! See #4964
